addons/sale/models/product_attribute_custom_value.py

```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models
+
+
+class ProductAttributeCustomValue(models.Model):
+    _inherit = "product.attribute.custom.value"
+
+    sale_order_line_id = fields.Many2one('sale.order.line', string="Sales Order Line", ondelete='cascade')
```

Comment:
This field should be removed. It has been replaced by the new res_model/res_id combo.

Reply:
Not really. For now the `sale` module does not inherit the mixin as we use a `product_description` field there.
The current problem is that `sale_order_line` use the field `name` for that purpose.
The solution for them to implement the mixin would be to rename the field (otherwise we would have a duplicate field) but this would lead to a sale-pocalypse just before the OXP.
So they are planning to implement the mixin in v17.

Reply:
You reasoning is correct though. The only problem I have here is that replacing the field will lead to add a duplicate create method in `sale_order_line` for something that will be removed in a few months. 
I guess it's better just to let it as it is and let them do the field `name` renaming and remove that file all at once in v17.